entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Set the environment
+if [ -z "$RAILS_ENV" ]
+then
+  export RAILS_ENV=production
+fi
+
+# Handle secrets based on env
+if [ "$RAILS_ENV" == "production" ]
+then
+  if [ -z "$SECRET_KEY_BASE" ]
+  then
+    echo "Setting environment variable \$SECRET_KEY_BASE."
```

Comment:
Do we need this `echo`? Who will read it?

Reply:
I was thinking this will stay in the logs and potentially help in the future, but happy to remove it!